apps/code-examples/src/app/code-examples/progress-indicator/passive-indicator/basic/demo.component.html

```diff
@@ -1,7 +1,13 @@
 <sky-progress-indicator [isPassive]="true" [startingIndex]="1">
-  <sky-progress-indicator-item title="Finished step"
+  <sky-progress-indicator-item
+    title="Finished step"
+    helpPopoverContent="Example help content for finished step"
+    helpPopoverTitle="Start here"
```

Comment:
Here, and in a few places, we need to ensure the inputs/outputs are alphabetized (especially in the code examples).

Reply:
reordered